test/CMakeLists.txt

```diff
@@ -23,11 +23,18 @@ if(MSVC)
   set(PCL_CTEST_ARGUMENTS ${PCL_CTEST_ARGUMENTS} -C $<$<CONFIG:Debug>:Debug>$<$<CONFIG:Release>:Release>)
 endif()
 
+# Enables you to disable visualization tests. Used on CI.
+if(PCL_DISABLE_VISUALIZATION_TESTS)
+  string(APPEND EXCLUDE_TESTS visualization)
+endif()
+
 # Enables you to disable GPU tests. Used on CI as it has no access to GPU hardware
 if(PCL_DISABLE_GPU_TESTS)
-  set(PCL_CTEST_ARGUMENTS ${PCL_CTEST_ARGUMENTS} -E gpu)
+  string(APPEND EXCLUDE_TESTS gpu)
 endif()
 
+set(PCL_CTEST_ARGUMENTS ${PCL_CTEST_ARGUMENTS} -E ${EXCLUDE_TESTS})
```

Comment:
`-E` is added even if no test is disabled. Also some quotation marks might be needed here to ensure correct whitespace. Do you know whether it has to be `-E gpu -E visualization` or `-E gpu visualization`? A quick internet search did not give any answers

Reply:
Good point. `-E regex` seems to be the API, so I'd guess it means `-E gpu` is correct, not `-E gpu viz`. I don't know if we can have 2 -E though

Reply:
I just know that before this [commit](https://github.com/PointCloudLibrary/pcl/pull/4426/commits/4e97364101c466e0814548309837e41e39c8110c), only the GPU was exluded and it failed running the visualization test. After the commit it correctly skipped both.
I can have a look at it again. I guess if its a regex, it should be possible to match one or the other, so they are correctly skipped.

Reply:
I think the bad documentation of CTest aids in exploration and exploitation of any behavior, as per Hyrum's law 

😆 If it works, I guess it's fine....

Reply:
I have updated to properly build the regex.